internal/pkg/featureflags/feature_flags.go

```diff
@@ -31,15 +30,18 @@ const (
 	userPath = "users/%s"
 )
 
-const cliProdEnv = "confluent.cloud"
+const (
+	ccloudUrl = "https://confluent.cloud"
+	devel     = "devel.cpdev.cloud"
+	stag      = "stag.cpdev.cloud"
```

Comment:
How come these don't start with https:// but `ccloudUrl` does?

Reply:
I'm going to change this. I was trying to get all of our launchdarkly code into the featureflags pkg, but that caused circular imports.